src/pages/Chat/startTyping.gql

```diff
@@ -0,0 +1,5 @@
+mutation startTyping($conversationId: Int!) {
```

Comment:
did you remove stopTyping ??

Reply:
Yes since UX-wise it seems to be fine without it. 
If we find it problematic (even without the sender-bias) we can introduce it.